.github/workflows/sonarcloud.yml

```
@@ -0,0 +1,31 @@
+          fetch-depth: 0
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@44454db4f0199b8b9685a5d763dc37cbf79108e1 # v2
+        with:
+          php-version: "8.3"
+          extensions: intl, mbstring, mysqli
+          tools: composer
+          coverage: xdebug
+
+      - name: SonarCloud Scan
+        if: ${{ secrets.SONAR_TOKEN != '' }}
```

actionlint (1.7.10)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)